### PR TITLE
Make Discord and Youtube links more concise in README

### PR DESCRIPTION
**Pull request by karthikmurali60**
closes https://github.com/EddieHubCommunity/support/issues/3686

#### What does this PR do?
This PR makes the Discord and Youtube links more concise in the README file

#### Is there any relevant issue to this PR?
https://github.com/EddieHubCommunity/support/issues/3686

### CODE REVIEW

**Review by TheRemyD — APPROVED** [review]
Links look great

File: README.md [hunk 1]
```diff
@@ -77,5 +77,5 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
 
 ## Socials
 
-Join our Discord community [here](http://discord.eddiehub.org)   
-Subscribe to our YouTube channel [here](https://www.youtube.com/user/eddiejaoude)
+Join our [Discord](http://discord.eddiehub.org) community
```
Comment (schmelto) [inline README.md:80]:
```suggestion
Join our [Discord community](http://discord.eddiehub.org)   
```

File: README.md [hunk 2]
```diff
@@ -77,5 +77,5 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
 
 ## Socials
 
-Join our Discord community [here](http://discord.eddiehub.org)   
-Subscribe to our YouTube channel [here](https://www.youtube.com/user/eddiejaoude)
+Join our [Discord](http://discord.eddiehub.org) community   
+Subscribe to our [YouTube](https://www.youtube.com/user/eddiejaoude) channel
```
Comment (schmelto) [inline README.md:81]:
```suggestion
Subscribe to our [YouTube channel](https://www.youtube.com/user/eddiejaoude)
```

**Review by schmelto — COMMENTED** [review]
Maybe this as small suggestion but not really needed I leave it to you if you want this styling change

File: README.md (hunk 1 above)
Comment (FahimFBA) [inline README.md:80]:
```suggestion
Join our [Discord community](http://discord.eddiehub.org).
```

File: README.md (hunk 2 above)
Comment (FahimFBA) [inline README.md:81]:
```suggestion
Subscribe to our [YouTube channel](https://www.youtube.com/user/eddiejaoude).
```

**Review by FahimFBA — CHANGES_REQUESTED** [review]

**Comment by FahimFBA** [comment]
Also, please correct the PR title.

```diff
- Make Discord and Youtube lniks more concise in README
+ Make Discord and Youtube links more concise in README
```

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by FahimFBA — APPROVED** [review]

**Comment by FahimFBA** [comment]
Everything looks perfect now.

**Comment by stemount** [comment]
Closing as there's PR duplication and this is an addition - nothing wrong with the changes! They are just not links right down at the bottom of the README.

**Pull request closed without merging**